src/key_command/constants.rs

```diff
@@ -8,8 +8,8 @@ macro_rules! cmd_constants {
             pub const $cmd_name: &str = $cmd_value;
         )*
 
-        pub fn commands() -> Vec<&'static str> {
-            vec![$($cmd_value,)*]
+        lazy_static! {
+            pub static ref COMMANDS: Vec<&'static str> = vec![$($cmd_name,)*];
```

Comment:
This isn't a necessary change, but this way the array gets initialized just once instead of on each call.

Reply:
Cool, I didn't know you could do that haha